src/Evolution/Triggers/SeparationLessThan.cpp

```diff
@@ -0,0 +1,78 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Evolution/Triggers/SeparationLessThan.hpp"
+
+#include <cstddef>
+#include <memory>
+#include <pup.h>
+#include <string>
+#include <unordered_map>
+
+#include "DataStructures/Tensor/EagerMath/Norms.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/Block.hpp"
+#include "Domain/Domain.hpp"
+#include "Domain/Structure/Element.hpp"
+#include "Domain/Structure/ElementId.hpp"
+
+namespace Triggers {
+SeparationLessThan::SeparationLessThan(const double separation)
+    : separation_(separation) {}
+
+bool SeparationLessThan::operator()(
+    const double time, const ::Domain<3>& domain, const Element<3>& element,
+    const std::unordered_map<
+        std::string, std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>>&
+        functions_of_time,
+    const tnsr::I<double, 3, Frame::Grid>& object_center_a,
+    const tnsr::I<double, 3, Frame::Grid>& object_center_b) const {
+  const ElementId<3>& element_id = element.id();
+  const size_t block_id = element_id.block_id();
+
+  const Block<3>& block = domain.blocks()[block_id];
+
+  tnsr::I<double, 3, Frame::Inertial> inertial_object_center_a{};
+  tnsr::I<double, 3, Frame::Inertial> inertial_object_center_b{};
+
+  if (block.has_distorted_frame()) {
```

Comment:
I think you don't need this because the grid-to-inertial map is available even when you have a distorted frame. Please double-check.

Reply:
Yeah but the Shape map isn't global, so it isn't valid inside the excision sphere. Only expansion and rotation are. So in a block with a distorted frame, we can't call the shape map, so we have to call the distorted to sphere map. However, all of this is mute with #5184.

Reply:
Ah ok, this should definitely be in a comment. Doesn't matter though if you'll refactor it with #5184 anyway.